Makefile

```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
 bin/installer:
 	@mkdir -p bin
 	go build -o bin/installer ./cmd/installer
+
+# make test-embed channel=Unstable app=slackernews
+.PHONY: test-embed
+test-emded: export OS=linux
+test-embed: export ARCH=amd64
+test-embed: static embedded-cluster
+	@echo "Cleaning up previous release directory..."
+	rm -rf ./hack/release
```

Comment:
We use the "build" directory in other places for gitignored artifacts

Reply:
ok, but this is not a build artifact, the build artifact is in the build dir   this is added to gitignore